data/mods/Backrooms/scenarios.json

```
@@ -9,6 +9,16 @@
     "start_name": "The Backrooms",
     "flags": [ "LONE_START" ]
   },
+  {
+    "type": "scenario",
+    "id": "backroom_dweller_starternpc",
+    "name": "Backrooms (with a starter npc)",
+    "points": 0,
+    "description": "What is this place?  You can't seem to find your way back out. But hey, at least you aren't alone. Will you cannibalize them as a last resort? Maybe get them to surgically implant bionics into you? Just keep them around for company? or task automation? Maybe you two will protect each-other? The possibilities have just widened, my friend. For you are no longer one. You are two, and that's better than what it used to be like.",
+    "allowed_locs": [ "sloc_backrooms" ],
+    "start_name": "The Backrooms",
+    "flags": [  ]
```

```suggestion
    "start_name": "The Backrooms"
```
Since there's no scenario flags, we don't need an empty array here.